generators/app/templates/scripts/_karma.conf.js

```diff
@@ -1,62 +1,107 @@
 /* eslint-disable no-console */
 const serveStatic = require('serve-static');
-const path = require('path');
-const serve = serveStatic(
-  path.join(__dirname, '..'),
-  {index: ['index.html', 'index.htm']}
-);
 
+/* allow static files to be served, note karma will takeover the /test directory */
 const StaticMiddlewareFactory = function(config) {
-  console.log(`**** Dev server started at http://${config.listenAddress}:${config.port}/ *****`);
+  console.log(`**** Karma static file server enabled in project root *****`);
```

Comment:
small change here, config.port would be wrong if the port is in-use and has to be incremented.